addon/components/labs-bbl-lookup.js

```diff
@@ -58,7 +58,7 @@ export default Component.extend({
       const validLot = this.get('validLot');
 
       if (validBlock && !validLot) {
-        const SQL = `SELECT the_geom FROM dof_dtm_block_centroids WHERE block= ${parseInt(block, 10)} AND borocode = ${code}`;
+        const SQL = `SELECT the_geom FROM dof_dtm_block_centroid WHERE block= ${parseInt(block, 10)} AND borocode = ${code}`;
```

Comment:
Should it be `dof_dtm_block_centroid**s**` plural?

Reply:
Yep, and once I fix that, this file shouldn't show up in the diff at all.